CHANGELOG.md

```diff
@@ -2,27 +2,32 @@
 
 # UNRELEASED
 
+### feat: add --open flag to dfx deploy
+
+The `--open` flag will open the browser to the canister's URL after deployment.
+
 ### feat: unify CLI options to specify arguments
 
 There are a few subcommands that take `--argument`/`--argument-file` options to set canister call/init arguments.
 
 We unify the related logic to provide consistent user experience.
- 
```

Comment:
Please revert all of the changes to this file that are not in the "UNRELEASED" section.

Reply:
Done!